### PR TITLE
Bugfix/cc 6193 adding unit tests

### PR DESCRIPTION
**Pull request by jacek-klapacz**
__Story / Bug id:__ https://bugzilla.stxnext.pl/show_bug.cgi?id=6193
__Project / App:__ wdrozenie
__Reference person:__ @JacekKlapacz
__Description:__

adding remaining unit tests 

__Results:__
pep8: empty

Report
======
139 statements analysed.

Statistics by type
------------------

+---------+-------+-----------+-----------+------------+---------+
|type     |number |old number |difference |%documented |%badname |
+=========+=======+===========+===========+============+=========+
|module   |6      |6          |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+
|class    |2      |2          |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+
|method   |9      |9          |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+
|function |12     |12         |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+

External dependencies
---------------------
::

    flask (presence_analyzer.main,presence_analyzer.views,presence_analyzer.utils)
    presence_analyzer 
      \-main (presence_analyzer.tests,presence_analyzer.views,presence_analyzer.utils)
      \-utils (presence_analyzer.tests,presence_analyzer.views)
      \-views (presence_analyzer.tests)

Raw metrics
-----------

+----------+-------+------+---------+-----------+
|type      |number |%     |previous |difference |
+==========+=======+======+=========+===========+
|code      |168    |50.60 |168      |=          |
+----------+-------+------+---------+-----------+
|docstring |101    |30.42 |101      |=          |
+----------+-------+------+---------+-----------+
|comment   |7      |2.11  |7        |=          |
+----------+-------+------+---------+-----------+
|empty     |56     |16.87 |56       |=          |
+----------+-------+------+---------+-----------+

Duplication
-----------

+-------------------------+------+---------+-----------+
|                         |now   |previous |difference |
+=========================+======+=========+===========+
|nb duplicated lines      |0     |0        |=          |
+-------------------------+------+---------+-----------+
|percent duplicated lines |0.000 |0.000    |=          |
+-------------------------+------+---------+-----------+

Messages by category
--------------------

+-----------+-------+---------+-----------+
|type       |number |previous |difference |
+===========+=======+=========+===========+
|convention |0      |0        |=          |
+-----------+-------+---------+-----------+
|refactor   |0      |0        |=          |
+-----------+-------+---------+-----------+
|warning    |1      |1        |=          |
+-----------+-------+---------+-----------+
|error      |0      |0        |=          |
+-----------+-------+---------+-----------+

% errors / warnings by module
-----------------------------

+------------------------+------+--------+---------+-----------+
|module                  |error |warning |refactor |convention |
+========================+======+========+=========+===========+
|presence_analyzer.tests |0.00  |100.00  |0.00     |0.00       |
+------------------------+------+--------+---------+-----------+

Messages
--------

+-----------------+------------+
|message id       |occurrences |
+=================+============+
|locally-disabled |5           |
+-----------------+------------+
|unused-import    |1           |
+-----------------+------------+

Global evaluation
-----------------
Your code has been rated at 9.93/10 (previous run: 9.93/10, +0.00)

__Migrations:__
N/A

__Rollback:__
N/A

__New imports / dependencies:__
N/A

__Unit Tests:__
presence-analyzer-jklapacz/src/presence_analyzer/tests.py

### CODE REVIEW

File: src/presence_analyzer/tests.py [hunk 1]
```diff
@@ -53,6 +53,38 @@ def test_api_users(self):
         self.assertEqual(len(data), 2)
         self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
 
+    def test_api_presence_weekday(self):
+        """
+        Test presence time of given user.
+        """
+        resp = self.client.get('/api/v1/presence_weekday/100')
+        self.assertEqual(resp.status_code, 404)
```
Comment (tomaszwozniak) [inline src/presence_analyzer/tests.py]: please split this, every case should be in new function

File: src/presence_analyzer/tests.py [hunk 2]
```diff
@@ -53,6 +53,38 @@ def test_api_users(self):
...
+        data = json.loads(resp.data)
+        self.assertEqual(len(data), 8)
+        self.assertEqual(data[0], ["Weekday", "Presence (s)"])
+        self.assertEqual(data[1], ["Mon", 0])
+
+    def test_api_mean_time_weekday(self):
+        """
+        Test mean presence time of given user.
+        """
+        resp = self.client.get('/api/v1/mean_time_weekday/100')
```
Comment (tomaszwozniak) [inline src/presence_analyzer/tests.py]: ditto

**Comment by tomaszwozniak** [comment]
minor comments

**Comment by jacek-klapacz** [comment]
__Story / Bug id:__ https://bugzilla.stxnext.pl/show_bug.cgi?id=6193
__Project / App:__ wdrozenie
__Reference person:__ @JacekKlapacz
__Description:__

split tests - every case in new function

__Results:__
pep8: empty
Report
======
1 statements analysed.

Statistics by type
------------------

+---------+-------+-----------+-----------+------------+---------+
|type     |number |old number |difference |%documented |%badname |
+=========+=======+===========+===========+============+=========+
|module   |1      |1          |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+
|class    |0      |0          |=          |0           |0        |
+---------+-------+-----------+-----------+------------+---------+
|method   |0      |0          |=          |0           |0        |
+---------+-------+-----------+-----------+------------+---------+
|function |0      |0          |=          |0           |0        |
+---------+-------+-----------+-----------+------------+---------+



Raw metrics
-----------

+----------+-------+------+---------+-----------+
|type      |number |%     |previous |difference |
+==========+=======+======+=========+===========+
|code      |0      |0.00  |0        |=          |
+----------+-------+------+---------+-----------+
|docstring |3      |50.00 |3        |=          |
+----------+-------+------+---------+-----------+
|comment   |1      |16.67 |1        |=          |
+----------+-------+------+---------+-----------+
|empty     |2      |33.33 |2        |=          |
+----------+-------+------+---------+-----------+



Duplication
-----------

+-------------------------+------+---------+-----------+
|                         |now   |previous |difference |
+=========================+======+=========+===========+
|nb duplicated lines      |0     |0        |=          |
+-------------------------+------+---------+-----------+
|percent duplicated lines |0.000 |0.000    |=          |
+-------------------------+------+---------+-----------+



Messages by category
--------------------

+-----------+-------+---------+-----------+
|type       |number |previous |difference |
+===========+=======+=========+===========+
|convention |0      |0        |=          |
+-----------+-------+---------+-----------+
|refactor   |0      |0        |=          |
+-----------+-------+---------+-----------+
|warning    |0      |0        |=          |
+-----------+-------+---------+-----------+
|error      |0      |0        |=          |
+-----------+-------+---------+-----------+



Global evaluation
-----------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00


__Migrations:__
N/A

__Rollback:__
N/A

__New imports / dependencies:__
N/A

__Unit Tests:__
presence_analyzer/tests.py .......
--------------------------------------------------------------------- coverage: platform linux2, python 2.7.10-final-0 ----------------------------------------------------------------------
Name                         Stmts   Miss  Cover   Missing
----------------------------------------------------------
presence_analyzer/__init__       1      0   100%   
presence_analyzer/helpers        1      1     0%   2
presence_analyzer/main           6      0   100%   
presence_analyzer/tests         67      5    93%   133-136, 140
presence_analyzer/utils         42      0   100%   
presence_analyzer/views         32      0   100%   
----------------------------------------------------------
TOTAL                          149      6    96%

**Pull request merged**